### DISCUSSION TITLE
Unable to install Spacy 2.1.0 with Python 3.9.1

### DISCUSSION THREAD

**Discussion by dimaelzein** [explosion/spaCy#7676] [Help: Installation] [answered]
I am unable to  install Spacy 2.1.0 with Python 3.9.1. with an error message "Could not find a version that satisfies..."
- Windows 10
- installing packaging from pip on cmd
- Recent versions of Spacy were working fine. But I have to downgrade to use neuralcoref 
- I already cleaned the environment and retried. Didn't work
- I already tried pip install spacy==2.1.0 --no-cache-dir. Didn't work

**Comment by adrianeboyd** [comment, marked as answer]
The latest version of python that this version was tested with was python 3.7. You'll probably have an easier time installing spacy and its dependencies with an earlier version of python, since there should be binary packages available for python 3.7.

**Reply by dimaelzein** [reply]
It worked! Many thanks !!
